content/about/notes/0.8.md

```diff
@@ -3,42 +3,26 @@ title: Istio 0.8
 
-- **Gateway for Ingress/Egress**. We no longer support combining Kubernetes Ingress specs with Istio route rules,
-as it has led to several bugs and reliability issues. Istio now supports a platform independent ingress/egress
-[Gateway](/docs/concepts/traffic-management/rules-configuration/#gateways) that works across
-Kubernetes and Cloud Foundry and integrates seamlessly with the routing rules.
+- **Streaming Envoy configuration**. By default Pilot now streams configuration to Envoy using its [ADS API](https://github.com/envoyproxy/data-plane-api/blob/master/XDS_PROTOCOL.md). This new approach increases effective scalability, reduces rollout delay and should eliminate spurious 404 errors.
 
-- **The new Gateway supports [Server Name Indication](https://en.wikipedia.org/wiki/Server_Name_Indication)** based routing,
-as well as serving a certificate based on the SNI value. HTTPS access to external services will be auto configured based on SNI.
-
-- **Envoy v2 APIs**. Users can choose to inject a sidecar using Envoy's v2 APIs. In this mode, Pilot uses [Envoy's v2 Aggregated Discovery Service
-API](https://www.envoyproxy.io/docs/envoy/latest/configuration/overview/v2_overview) to push configuration to the data plane.
-This new approach increases scalability of the control plane.
+- **Gateway for Ingress/Egress**. We no longer support combining Kubernetes Ingress specs with Istio routing rules as it has led to several bugs and reliability issues. Istio now supports a platform independent [Gateway](/docs/concepts/traffic-management/rules-configuration/#gateways) model for ingress & egress proxies that works across Kubernetes and Cloud Foundry and works seamlessly with routing. The Gateway supports [Server Name Indication](https://en.wikipedia.org/wiki/Server_Name_Indication) based routing,
+as well as serving a certificate based on the server name presented by the client.
 
 - **Constrained Inbound Ports**. We now restrict the inbound ports in a pod to the ones declared by the apps running inside that pod.
 
 ## Security
 
-- **Introducing Citadel**. We've finally given a name to our security component. What was
-formerly known as Istio-Auth or Istio-CA is now called Citadel.
+- **Introducing Citadel**. We've finally given a name to our security component. What was formerly known as Istio-Auth or Istio-CA is now called Citadel.
 
-- **Multicluster Support**. We support per-cluster Citadel in multicluster deployments such that all Citadels share the same root cert
-and workloads can authenticate each other across the mesh.
+- **Multicluster Support**. We support per-cluster Citadel in multicluster deployments such that all Citadels share the same root certificate and workloads can authenticate each other across the mesh.
```

Comment:
How about add a new section for `Multicluster` support? As this is new alpha feature in 0.8 not limited in security.